src/guide/render-function.md

```diff
@@ -394,8 +394,10 @@ render() {
 If we're writing a lot of `render` functions, it might feel painful to write something like this:
 
 ```js
+import { resolveComponent } from 'vue'
+
 Vue.h(
-  'anchored-heading',
+  resolveComponent('anchored-heading'),
```

Comment:
There seems to be a small inconsistency here. The rest of this example is using `Vue.h` but this line uses an imported `resolveComponent` rather than `Vue.resolveComponent`.